pyspark/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/context.py

```diff
@@ -32,12 +32,12 @@ def init_spark_session(self) -> None:
         """
 
         # Load the spark configuration in spark.yaml using the config loader
-        parameters = self.config_loader.get("spark*", "spark*/**")
+        parameters = self._config_loader.get("spark*", "spark*/**")
         spark_conf = SparkConf().setAll(parameters.items())
 
         # Initialise the spark session
         spark_session_conf = (
-            SparkSession.builder.appName(self.package_name)
+            SparkSession.builder.appName(self._package_name)
```

Comment:
If `self.package_name` is not available as an attribute, we may have to fix this in the doc too, as I found we are using `self._config_loader`, but `self.package_name` there.

https://kedro.readthedocs.io/en/stable/tools_integration/pyspark.html#:~:text=from%20typing%20import,sparkContext.setLogLevel(%22WARN%22)

Reply:
Just did a global search on `kedro` and `kedro_starter`, seems it is only used in Spark related doc, so probably we just have to fix the `kedro` doc for this one.